homeassistant/components/overkiz/entity.py

```diff
@@ -114,7 +115,12 @@ def __init__(
         """Initialize the device."""
         super().__init__(device_url, coordinator)
         self.entity_description = description
-        self._attr_unique_id = f"{super().unique_id}-{self.entity_description.key}"
+
+        # Keep ids stable for pyoverkiz >= 1.10.1 where enums changed to StrEnum.
```

Comment:
I think the comment is wrong, shouldn't it be like this?
```suggestion
        # Keep ids stable for pyoverkiz >= 1.10.0 where enums changed to StrEnum.
```

Reply:
You're right. This needs to be `1.10.0` here and in the commit message. There's a chance that github would lose the discussion below if I change the commit in the PR. I'd therefore only do that in a later step if we go on with this PR.